### ISSUE TITLE
git repo url in layman.xml needs to be changed from git:// to https://

### ISSUE THREAD

**Issue by jeandestouches** [hhoffstaette/portage#7]
Hi,
When syncing your repo this morning, this message appeared :

    The unauthenticated git protocol on port 9418 is no longer supported.
    Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information

I use eselect repository to manage the extra repos but I guess this applies to layman.xml if one use it as well.
So this probably needs to be changed from : 

    <source type="git">git://github.com/hhoffstaette/portage.git</source>

to

    <source type="git">https://github.com/hhoffstaette/portage.git</source>

Cheers

**Comment by hhoffstaette** [comment]
Fixed in 4e47b7bc - thank you!
